v2/yarpcthrift/codec_test.go

```diff
@@ -53,7 +53,7 @@ func TestDecodeAgnosticProto(t *testing.T) {
 	// calling Decode again should fail, because it attempts to set a new
 	// responder.
 	res, err = testCodec.Decode(reqBuf)
-	assert.Error(t, err, "unexpected error")
+	assert.EqualError(t, err, "code:internal message:tried to overwrite a responder for thrift codec")
```

Comment:
We could also assert that this is an internal error code by using `yarpcerror.FromError(err).Code()`